src/app/lib/button/button-demo/button-demo.component.html

```diff
@@ -1,75 +1,83 @@
 <div class="demo-page">
-  <h2 style="margin: 0">Buttons</h2>
-  <h4 style="font-weight: 300; margin-top: 0">Buttons communicate the action that will occur when the user touches them.</h4>
-  <h5>
-    Primary Buttons
-  </h5>
-  <div class="sample-buttons">
-    <hc-button [options]="primaryButton1Options"></hc-button>
-    <hc-button [options]="primaryButton2Options"></hc-button>
-    <hc-button [options]="primaryButton3Options"></hc-button>
-    <hc-button [options]="primaryButton4Options"></hc-button>
-  </div>
-  <h5>
-    Secondary Buttons
-  </h5>
-  <hc-button [options]="secondaryButtonOptions"></hc-button>
-  <h5>
-    Disabled Buttons
-  </h5>
-  <hc-button [options]="disabledButtonOptions"></hc-button>
-  <h5>
-    Tertiary Buttons
-  </h5>
-  <hc-button [options]="tertiaryButtonOptions"></hc-button>
-  <br>
-  <br>
-  <hr style="height:1px;border:none;background-color:#E5E5E5;">
-  <br>
+    <h2 style="margin: 0">Buttons</h2>
+    <h4 style="font-weight: 300; margin-top: 0">Buttons communicate the action that will occur when the user touches
+        them.</h4>
+    <h5>
+        Primary Buttons
+    </h5>
+    <div class="sample-buttons">
+        <button hc-button title="Primary Button" color="primary">Primary Button</button>
+        <button hc-button title="Primary-Alt1 Button" color="primary-alt1">Primary Alt1 Button</button>
+        <button hc-button title="Primary-Alt2 Button" color="primary-alt2">Primary Alt2 Button</button>
+        <button hc-button title="Primary-Alt3 Button" color="primary-alt3">Primary Alt3 Button</button>
+    </div>
+    <h5>
+        Secondary Buttons
+    </h5>
+    <button hc-button title="Secondary Button" color="secondary">Secondary Button</button>
+    <h5>
+        Disabled Buttons
+    </h5>
+    <button hc-button title="Disabled Button" color="primary" disabled>Disabled Button</button>
```

Comment:
Love how semantic the markup is and how you can use the native apis like disabled. Very nice.